src/main/java/uk/gov/hmcts/reform/em/stitching/pdf/PDFUtility.java

```diff
@@ -102,14 +118,54 @@ public static void addLink(PDDocument document, PDPage from, PDPage to, String t
         link.setRectangle(rectangle);
         from.getAnnotations().add(link);
 
+        PDBorderStyleDictionary borderLine = new PDBorderStyleDictionary();
+        borderLine.setStyle(PDBorderStyleDictionary.STYLE_UNDERLINE);
+        borderLine.setWidth(0);
+        link.setBorderStyle(borderLine);
+
         addText(document, from, text, xxOffset + 5, yyOffset - 3, font, fontSize);
     }
 
+    public static void addLink(PDDocument document, PDPage from, PDPage to, String text, float yyOffset,
+                               PDType1Font font, int fontSize, PDOutlineItem subtitle) throws IOException {
+        addLink(document, from, to, text, yyOffset, 45, font, fontSize, subtitle);
+    }
+
+    public static void addLink(PDDocument document, PDPage from, PDPage to, String text, float yyOffset, float xxOffset,
+                               PDType1Font font, int fontSize, PDOutlineItem subtitle) throws IOException {
```

Comment:
what is this method for? it doesn't appear to use the subtitle

Reply:
This method is to add links for documents which dont have any outline and part of the Bundle.

Reply:
I have made some some changes and removed this extra parameter, it was is resolved as part of the Sonar fix now .